examples/03-refresh-auth-token.ts

```diff
@@ -8,82 +8,82 @@
 /**
  * Get refresh token for a guest user
  * Usage: ts-node examples/03-refresh-auth-token.ts
- * Note: Replace configuration parameters before running
+ * For more information, see (Shopper Login and API Access Service)[https://developer.commercecloud.com/s/api-details/a003k00000VWfNDAA1/commerce-cloud-developer-centershopperloginandapiaccessservice].
```

Comment:
```suggestion
 * For more information, see [Shopper Login and API Access Service](https://developer.commercecloud.com/s/api-details/a003k00000VWfNDAA1/commerce-cloud-developer-centershopperloginandapiaccessservice).
```
`[text](link)` is for markdown, but I don't know if this gets converted to markdown anywhere. 🤔